api/models/vpp/interfaces/span.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package vpp.interfaces;
+
+option go_package = "github.com/ligato/vpp-agent/api/models/vpp/interfaces;vpp_interfaces";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+message Span {
+    enum State {
```

Comment:
Wouldn't it be better to call this parameter "Direction"?  "State" sounds like something related to runtime status as opposed to configuration.